flocker/node/test/test_deploy.py

```diff
@@ -619,7 +619,9 @@ def test_does_not_exist(self):
     name=APP_NAME2,
     image=DockerImage.from_string(UNIT_FOR_APP2.container_image)
 )
-EMPTY_NODESTATE = NodeState(hostname=u"example.com")
+EMPTY_NODESTATE = NodeState(hostname=u"example.com",
+                            manifestations={}, applications=[],
+                            used_ports=[], devices={}, paths={})
```

Comment:
Kind of a nit but grouping application attributes together and grouping dataset attributes together wouldn't hurt.